frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_service.dart

```diff
@@ -1,24 +1,22 @@
-import 'package:appflowy/plugins/database_view/application/field/field_info.dart';
+import 'package:appflowy_backend/protobuf/flowy-database2/cell_entities.pb.dart';
 import 'package:appflowy_backend/protobuf/flowy-database2/database_entities.pb.dart';
+import 'package:appflowy_backend/protobuf/flowy-database2/select_option.pb.dart';
 import 'package:dartz/dartz.dart';
 import 'package:appflowy_backend/dispatch/dispatch.dart';
 import 'package:appflowy_backend/protobuf/flowy-error/errors.pb.dart';
 import 'package:appflowy_backend/protobuf/flowy-database2/field_entities.pb.dart';
-import 'package:freezed_annotation/freezed_annotation.dart';
-
-part 'field_service.freezed.dart';
 
 /// FieldService consists of lots of event functions. We define the events in the backend(Rust),
 /// you can find the corresponding event implementation in event_map.rs of the corresponding crate.
 ///
 /// You could check out the rust-lib/flowy-database/event_map.rs for more information.
 class FieldBackendService {
```

Comment:
Making the methods of `FieldBackendService` static would require each method to accept `viewid` and `fieldid` as parameters. Additionally, the static methods appear similar to event methods. 

Both are work for me.

Reply:
Yes I changed it so that I don't have to pass the field controller everywhere, but now I have to pass the viewId everywhere. I was considering `context.read<Grid/Board/CalendarBloc>.databaseController.fieldController.backendService` as well, but we'd have to do inheritance on the bloc and looking for the ancestor is slightly slower